Text-Code/text-to-code/code/run.py

```diff
@@ -375,7 +375,9 @@ def eval_bleu(args, model, tokenizer, file_type='test', num=2000):
             p = []       
             zero = torch.cuda.LongTensor(1).fill_(0)
             for i in range(inputs.shape[0]):
-                past_hidden = [x[:, i:i+1].expand(-1, beam_size, -1, -1, -1) for x in outputs]
+                # Compatible with transformers version 3.3.0 and 4.13.0
+                past = [torch.cat([x[0].unsqueeze(0),x[1].unsqueeze(0)],dim=0) if type(x)==tuple else x for x in outputs]
+                past_hidden = [x.data.index_select(1, beam.getCurrentOrigin()) for x in past]
```

Comment:
`beam` has not been assigned here

Reply:
I forgot to make two different changes, and now I has correct one of them.